nima_core/dream_consolidation.py

```
@@ -0,0 +1,542 @@
+                seen_similar.add(m1["id"])
+                seen_similar.add(m2["id"])
+                patterns.append({
+                    "type": "near_duplicate",
+                    "ids": [m1["id"], m2["id"]],
+                    "similarity": round(_simple_similarity(t1, t2), 2),
+                })
+
+    return patterns
+
+
+def _synthesize_insights(patterns: List[Dict], memories: List[Dict],
```

Ruff (0.15.1)

[warning] Unused function argument: `memories`

(ARG001)